### PR TITLE
Feat(Bounds): add new methods for 2 missing corners

### PR DESCRIPTION
**Pull request by ghybs**
As suggested in https://github.com/Leaflet/Leaflet/issues/5475, this PR adds `getTopLeft()` and `getBottomRight()` methods to `Bounds` to retrieve the 2 missing corners (besides `getBottomLeft()` and `getTopRight()` already existing).

Having now the 4 corners make `Bounds` more in line with `LatLngBounds`.

Also added 4 tests (for each of the corner retrieval methods).

Closes https://github.com/Leaflet/Leaflet/issues/5475 together with the previous PR https://github.com/Leaflet/Leaflet/pull/5487

### CODE REVIEW

File: src/geometry/Bounds.js [hunk 1]
```diff
@@ -73,6 +73,18 @@ Bounds.prototype = {
...
+	// @method getTopLeft(): Point
+	// Returns the top-left point of the bounds.
+	getTopLeft: function () {
+		return new Point(this.min.x, this.min.y); // left, top
+	},
+
+	// @method getBottomRight(): Point
+	// Returns the bottom-right point of the bounds.
+	getBottomRight: function () {
+		return new Point(this.max.x, this.max.y); // right, bottom
```
Comment (mourner) [inline src/geometry/Bounds.js]: You can write it as `return this.max.clone();`.

Comment (ghybs) [inline src/geometry/Bounds.js]: Hi,

Thanks for the review and catch!

Now that I realize (:ashamed:) that we just return the same values  as `this.max`, I wonder if we should even clone it at all?

`LatLngBounds` do not bother cloning: https://github.com/Leaflet/Leaflet/blob/master/src/geo/LatLngBounds.js#L100-L110

Cloning is the exact transcription of my initial code.
Doing so is for sure the safest way, but slightly expensive performance-wise (e.g. see https://github.com/Leaflet/Leaflet.markercluster/pull/536#issuecomment-149535377).

Not cloning is the most simple, but could lead to app mistakes if the developer modifies them without cloning first.
But it is entirely consistent with `LatLngBounds`.

Furthermore, as these 2 methods are new, it should not break anything (yet) exposing them with potentially error-prone behaviour, should the app developer be not cautious enough.

For simplicity and consistency with `LatLngBounds`, I assume you would be ok to not clone and directly output `this.max` (same for `this.min`), so I will modify this PR accordingly.

Please feel free to comment back should you have a different position.

**Comment by ghybs** [comment]
Took the opportunity to add a documentation comment that the 2 new methods are just shorthands for the already available properties `this.min` and `this.max`, with links.
